docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -32,18 +31,28 @@ TIP: To sort the rules list, click any column header. To sort in descending orde
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on its name in the table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Installed Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+For information about rule execution gaps (which are periods of time when a rule didn't run as expected), use the panel above the table. The time filter on the left allows you to select a time range for viewing gap data. The **Total rules with gaps:** field tells you how many rules have unfilled or partially filled gaps within the selected time range. The **Only rules with gaps** filter on the right lets you only display rules with unfilled or partially filled gaps. 
```

Comment:
> a rule didn't run as expected  


It's not super clear what this mean. When rule has error - we can say that it's didn't run as excepted, but there will be no gap.

Reply:
Is the phrase "as expected" the part that's not quite accurate? I mainly added it because my understanding of the gaps feature is that it helps users monitor and fix interruptions (planned and unplanned) in rule executions.

Is this more accurate instead?

```suggestion
For information about rule execution gaps (which are periods of time when a rule didn't run), use the panel above the table. The time filter on the left allows you to select a time range for viewing gap data. The **Total rules with gaps:** field tells you how many rules have unfilled or partially filled gaps within the selected time range. The **Only rules with gaps** filter on the right lets you only display rules with unfilled or partially filled gaps. 
```

Reply:
I think we can say "information about rule execution gaps (which are periods of time when a rule didn't run)" - it is true, and simpler.
We do plan to make it more refined in the future, but for now this would be accurate